src/chip/chip.svelte

```diff
@@ -0,0 +1,15 @@
+<script>
+  import classes from '../_utils/classes.js';
+
+  let _class = null;
+  export { _class as class };
+  export let small = false;
+  export let outline = false;
+  export let noPadding = false;
```

Comment:
I feel like allowing to specify a custom padding value would be more flexible
```suggestion
  export let padding = 0;
```
and then use in `style="padding: {padding}px;"` or something similar

Reply:
I'm psychologically against inline styles. Ew-ew

Reply:
Well, since Svelte doesn't support using JS variables in CSS, I see no other way.

Reply:
Except for the way this is built now) one can still style the cards with CSS